src/test/kotlin/minesweeper/CordsTest.kt

```diff
@@ -0,0 +1,56 @@
+package minesweeper
+
+import io.kotest.core.spec.style.ExpectSpec
+import io.kotest.matchers.collections.shouldContainAll
+import io.kotest.matchers.shouldBe
+import minesweeper.domain.MapCord
+import minesweeper.domain.MapCords
+
+/**
+ * @see MapCords
+ */
+class CordsTest : ExpectSpec({
+
+    expect("높이와 너비를 입력하면 해당 크기 만큼의 좌표 묶음을 생성한다") {
```

Comment:
```suggestion
    expect("높이와 너비를 입력하면 높이와 너비를 곱한만큼의 좌표 묶음을 생성한다") {
```
"해당" 이라는 문구보다는 조금 더 구체적으로 작성해보면 어떨까요?